packages/DropdownMenu/src/DropdownMenu.js

```diff
@@ -11,32 +10,37 @@ import Item from "./components/Item";
 import { contentStyles } from "./DropdownMenu.styles";
 
 const propTypes = {
-  /** Alignment of the dropdown menu */
-  align: PropTypes.oneOf(AlignTypes.ALL),
+  /** Alignment of the Popover */
+  align: Popover.propTypes.align,
```

Comment:
`eslint` warns me about this, but it seems like a good way to keep these propTypes in sync.

Reply:
Yeah the warning is for when you remove proptypes from production builds. In a certain configuration of that babel plugin, this line would start throwing.
We don't do that, but it'd be interesting to see what impact on the bundle size that would have.

Reply:
If propTypes are removed, then this would be removed too, so it wouldn't throw an error? 
If an error is thrown in a forest that doesn't exist, does anyone hear the build fail? 🤔

Reply:
Oh I think you're right 🤦 I'm not too familiar with situations like this.